dub.sdl

```diff
@@ -0,0 +1,63 @@
+name "dmd"
+description "The DMD compiler"
+authors "Walter Bright"
+copyright "Copyright © 1999-2017, Digital Mars"
+license "BSL-1.0"
+
+targetType "library"
+
+// This is required because otherwise Dub will include all source files in src.
+// Just picked an existing directory which doesn't contain any D source files.
+sourcePaths "docs"
+
+// This is required because Dub needs to have at lease one source file
+sourceFiles "src/ddmd/package.d"
```

Comment:
Although it seems like it can still be used as a dependency without any problems. But the natural `dub build` doesn't work anymore.